schlib/rules/rule4_9.py

```diff
@@ -8,106 +8,127 @@ class Rule(KLCRule):
     """
     def __init__(self, component):
         super(Rule, self).__init__(component, 'Rule 4.9 - Default fields', 'Default fields contain the correct information')
-        
+
     def checkVisibility(self, field):
         return field['visibility'] == 'V'
-        
+
+    # return True if a field is empty else false
+    def checkEmpty(self, field):
```

Comment:
I don't really understand that part of the code (this function) ... what does it do?
At least for me the scripts in this PR always report KLC-violation 4.9 :
```
Checking symbol 'Pololu_Breakout_DRV8825':
  Violating Rule 4.9 - Default fields
    Default fields contain the correct information
    Datasheet field must be EMPTY
```